src/change_stream.ts

```diff
@@ -107,94 +108,220 @@ export interface PipeOptions {
  * @public
  */
 export interface ChangeStreamOptions extends AggregateOptions {
-  /** Allowed values: 'updateLookup'. When set to 'updateLookup', the change stream will include both a delta describing the changes to the document, as well as a copy of the entire document that was changed from some time after the change occurred. */
+  /**
+   * Allowed values: 'updateLookup'. When set to 'updateLookup',
+   * the change stream will include both a delta describing the changes to the document,
+   * as well as a copy of the entire document that was changed from some time after the change occurred.
+   */
   fullDocument?: string;
   /** The maximum amount of time for the server to wait on new documents to satisfy a change stream query. */
   maxAwaitTimeMS?: number;
-  /** Allows you to start a changeStream after a specified event. See {@link https://docs.mongodb.com/manual/changeStreams/#resumeafter-for-change-streams|ChangeStream documentation}. */
+  /**
+   * Allows you to start a changeStream after a specified event.
+   * @see https://docs.mongodb.com/manual/changeStreams/#resumeafter-for-change-streams
+   */
   resumeAfter?: ResumeToken;
-  /** Similar to resumeAfter, but will allow you to start after an invalidated event. See {@link https://docs.mongodb.com/manual/changeStreams/#startafter-for-change-streams|ChangeStream documentation}. */
+  /**
+   * Similar to resumeAfter, but will allow you to start after an invalidated event.
+   * @see https://docs.mongodb.com/manual/changeStreams/#startafter-for-change-streams
+   */
   startAfter?: ResumeToken;
   /** Will start the changeStream after the specified operationTime. */
   startAtOperationTime?: OperationTime;
-  /** The number of documents to return per batch. See {@link https://docs.mongodb.com/manual/reference/command/aggregate|aggregation documentation}. */
+  /**
+   * The number of documents to return per batch.
+   * @see https://docs.mongodb.com/manual/reference/command/aggregate
+   */
   batchSize?: number;
 }
 
 /** @public */
-export interface ChangeStreamDocument<TSchema extends Document = Document> {
+export interface ChangeStreamFullNameSpace {
+  /** The namespace */
+  ns: { db: string; coll: string };
```

Comment:
I notice further down in the `rename` interface, you reuse `{ db: string; coll: string }`.  Ultimately, `{ db: string; coll: string }` is the namespace.  Could we create the type as
```typescript
interface ChangeStreamFullNamespace { db: string; coll: string }
```
so we can reuse it where needed?  Then we can add `ns: ChangeStreamFullNamespace` to the `ChangeStreamDocumentCommon` object.

Reply:
Changed this, we can't add it to common because it's missing from a number of the events